padrino-helpers/lib/padrino/rendering/erubi_template.rb

```diff
@@ -0,0 +1,43 @@
+module Padrino
+  module Rendering
+    module Erubi
+      module SafeBufferEnhancer
+        def add_expression_result(code)
+          @src << " @__in_ruby_literal = true; #{bufvar}.concat((" << code << ').to_s); @__in_ruby_literal = false;'
+        end
+
+        def add_expression_result_escaped(code)
+          @src << " #{bufvar}.safe_concat #{@escapefunc}((" << code << "));"
+        end
+
+        def add_text(text)
+          @src << " #{bufvar}.safe_concat '" << escape_text(text) << "';" unless text.empty?
+        end
+
+        def escape_text(text)
```

Comment:
The #escape_text definition was argued against by Jeremy because it's a local escaping of a local single quote.

https://github.com/jeremyevans/erubi/blob/cb3e65a209aea4a540acc7d7e3bc9b4554a87180/lib/erubi.rb#L170-L172